src/sql/workbench/electron-browser/modelComponents/queryTextEditor.ts

```diff
@@ -132,9 +132,9 @@ export class QueryTextEditor extends BaseTextEditor {
 
 	public setHeightToScrollHeight(configChanged?: boolean): void {
 		let editorWidget = this.getControl() as ICodeEditor;
-		this._config = new Configuration(true, undefined, editorWidget.getDomNode(), this.accessibilityService);
+		let layoutInfo = editorWidget.getLayoutInfo();
 		if (!this._scrollbarHeight) {
-			this._scrollbarHeight = this._config.editor.viewInfo.scrollbar.horizontalScrollbarSize;
+			this._scrollbarHeight = editorWidget.getLayoutInfo().horizontalScrollbarHeight;
```

Comment:
Shouldn't you be using the `layoutInfo` object here?

Reply:
Nice catch!